cmd/check_operator_test.go

```diff
@@ -15,6 +15,7 @@ import (
 
 var _ = Describe("Check Operator", func() {
 	Context("when running the check operator subcommand", func() {
+		BeforeEach(artifactsAndLogTemp)
```

Comment:
SUPER NIT: Can this function be renamed to declare exactly what it's doing?

Reply:
Sure. It was a quick name. It's not the most descriptive.